ios/FluentUI/Pill Button Bar/PillButton.swift

```diff
@@ -203,19 +121,36 @@ open class PillButton: UIButton {
         if let window = window {
             if isSelected {
                 if isEnabled {
-                    backgroundColor = customSelectedBackgroundColor ?? style.selectedBackgroundColor(for: window)
-                    setTitleColor(customSelectedTextColor ?? style.selectedTitleColor(for: window), for: .normal)
+                    if let customSelectedBackgroundColor = customSelectedBackgroundColor {
+                        backgroundColor = customSelectedBackgroundColor
+                    } else {
+                        backgroundColor = isHighlighted
+                            ? PillButton.selectedHighlightedBackgroundColor(for: window, for: style)
+                            : PillButton.selectedBackgroundColor(for: window, for: style)
+                    }
+
+                    setTitleColor(customSelectedTextColor ?? PillButton.selectedTitleColor(for: window, for: style), for: .normal)
+                    setTitleColor(customSelectedTextColor ?? PillButton.selectedHighlightedTitleColor(for: window, for: style), for: .highlighted)
                 } else {
-                    backgroundColor = style.selectedDisabledBackgroundColor(for: window)
-                    setTitleColor(style.selectedDisabledTitleColor(for: window), for: .normal)
+                    backgroundColor = PillButton.selectedDisabledBackgroundColor(for: window, for: style)
```

Comment:
Do we need to specify PillButton on these extension calls?
**PillButton**.selectedDisabledBackgroundColor(for: window, for: style)

Reply:
**PillButton** is needed because the color extension calls are made static.